ovirt/ovirt_vm_optimize_cpu_settings.go

```diff
@@ -0,0 +1,70 @@
+package ovirt
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	ovirtclient "github.com/ovirt/go-ovirt-client"
+)
+
+var vmOptimizeCPUSettingsSchema = map[string]*schema.Schema{
+	"id": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "oVirt ID of the VM to be started.",
+	},
+	"vm_id": {
+		Type:             schema.TypeString,
+		Required:         true,
+		Description:      "oVirt ID of the VM to be optimized.",
+		ForceNew:         true,
+		ValidateDiagFunc: validateUUID,
+	},
+}
+
+func (p *provider) vmOptimizeCPUSettingsResource() *schema.Resource {
+	return &schema.Resource{
```

Comment:
I think `UpdateContext` this resource is missing. But I am not sure if it is required here.

Reply:
If all parameteres are `ForceNew: true` then you don't need an update function. Terraform will complain if you miss it.